### PR TITLE
docs(plugins) Correct tag

### PR DESCRIPTION
**Pull request by EugeneHlushko**
Use real tag as opposed to the next similar reference to `<link>` tag...

### CODE REVIEW

File: src/content/plugins/html-webpack-plugin.md [hunk 1]
```diff
@@ -51,12 +52,9 @@ This will generate a file `dist/index.html` containing the following:
 </html>
 ```
 
-If you have multiple webpack entry points, they will all be included with `script`
-tags in the generated HTML.
+If you have multiple webpack entry points, they will all be included with `<script>` tags in the generated HTML.
```
Comment (pranshuchittora) [inline src/content/plugins/html-webpack-plugin.md]: **in** the generated HTML -> **inside** the generated HTML

File: src/content/plugins/html-webpack-plugin.md [hunk 2]
```diff
@@ -51,12 +52,9 @@ This will generate a file `dist/index.html` containing the following:
...
 
-If you have any CSS assets in webpack's output (for example, CSS extracted
-with the [MiniCssExtractPlugin](/plugins/mini-css-extract-plugin/))
-then these will be included with `<link>` tags in the HTML head.
+If you have any CSS assets in webpack's output (for example, CSS extracted with the [MiniCssExtractPlugin](/plugins/mini-css-extract-plugin/)) then these will be included with `<link>` tags in the HTML head.
```
Comment (pranshuchittora) [inline src/content/plugins/html-webpack-plugin.md]: If you have any CSS assets in webpack's output (for example, CSS extracted with the [MiniCssExtractPlugin](/plugins/mini-css-extract-plugin/)) then those will be included with `<link>` tags inside the `<head>` of HTML.

File: src/content/plugins/html-webpack-plugin.md (hunk 1 above)
Comment (EugeneHlushko) [reply src/content/plugins/html-webpack-plugin.md]: I would keep **in**, why change?

File: src/content/plugins/html-webpack-plugin.md (hunk 2 above)
Comment (EugeneHlushko) [reply src/content/plugins/html-webpack-plugin.md]: i dont see a benefit of changing **in** to **inside**, but i like the idea of the rephrasing

File: src/content/plugins/html-webpack-plugin.md (hunk 1 above)
Comment (pranshuchittora) [reply src/content/plugins/html-webpack-plugin.md]: Hmm... Ok

**Comment by montogeek** [comment]
Thanks!

**Comment by pranshuchittora** [comment]
Awesome 🚀🤘

**Pull request merged**